presto-tests/src/test/java/com/facebook/presto/tests/TestLocalBinarySpilledQueries.java

```diff
@@ -66,4 +67,15 @@ private static LocalQueryRunner createLocalQueryRunner()
 
         return localQueryRunner;
     }
+
+    @Override
+    @Test(expectedExceptions = IllegalArgumentException.class)
```

Comment:
please do not use expectedException, use assertQueryFails instead. 
Why does it fail with IAE? Semantic exception would be much better.